src/BirdLIST.js

```diff
@@ -0,0 +1,38 @@
+// import { useState } from "react";
+
+
+function BirdLIST({birdArray,birdCostArray}){
```

Comment:
Remember to use the autoformatter in VSCode, it'll help out a lot with legibility.